nipype/interfaces/r.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""Interfaces to run R scripts."""
+import os
+
+from .. import config
+from .base import (
+    CommandLineInputSpec,
+    InputMultiPath,
+    isdefined,
+    CommandLine,
+    traits,
+    File,
+    Directory,
+)
+
+
+def get_r_command():
+    if "NIPYPE_NO_R" in os.environ:
+        return None
+    r_cmd = os.getenv("RCMD", default="R")
+    
+    return r_cmd if which(r_cmd) else None
+
+
+no_r = get_r_command() is None
+
+
+class RInputSpec(CommandLineInputSpec):
+    """ Basic expected inputs to R interface """
+
+    script = traits.Str(
+        argstr='-e "%s"', desc="R code to run", mandatory=True, position=-1
+    )
+    # non-commandline options
+    rfile = traits.Bool(True, desc="Run R using R script", usedefault=True)
+    script_file = File(
+        "pyscript.R", usedefault=True, desc="Name of file to write R code to"
+    )
+    
+
+class RCommand(CommandLine):
+    """Interface that runs R code
+
+    >>> import nipype.interfaces.r as r
+    >>> r = r.RCommand(rfile=False)  # don't write script file
+    >>> r.inputs.script = "Sys.getenv('USER')"
+    >>> out = r.run()  # doctest: +SKIP
+    """
+
+    _cmd = get_r_command()
+    _default_r_cmd = None
```

Comment:
Admittedly, I wrote the entire R interface and tests by copying the matlab implementation and find+replacing matlab with R. So it's done that way because that's the way the matlab interface works, but I agree it's extraneous and removed it (would we want to remove it from the matlab interface as well for consistency's sake?)

Reply:
I'm inclined to leave it alone in MATLAB, since it's been there so long. Somebody may use it for something, and we'd be breaking things for them...